### PR TITLE
PluginManager normalizations

### PR DESCRIPTION
**Pull request by mjauvin**
replaces #4837 

fix isDisabled() to be case insensitive
fix findByIdentifier() to be case insensitive

### CODE REVIEW

**Comment by mjauvin** [comment]
This is in line with earlier improvements made in #4337

**Comment by mjauvin** [comment]
bump!

File: modules/system/classes/PluginManager.php [hunk 1]
```diff
@@ -348,7 +348,8 @@ public function findByNamespace($namespace)
     public function findByIdentifier($identifier)
     {
         if (!isset($this->plugins[$identifier])) {
-            $identifier = $this->normalizeIdentifier($identifier);
+            $code = $this->getIdentifier($identifier);
+            $identifier = $this->normalizeIdentifier($code);
```
Comment (LukeTowers) [inline modules/system/classes/PluginManager.php:352]: Why is the getIdentifier() call required here?

Comment (mjauvin) [reply modules/system/classes/PluginManager.php:352]: @LukeTowers I understand the reason you ask the question as it doesn't seem to make sense considering the parameter names... we might want to rewrite part of this class' methods at some point, but I was just trying to emulate what the isDisabled() method does with its identifier.

Am I doing it wrong here, please feel free to let me know.

**Pull request merged**